.github/workflows/scorecards.yml

```
@@ -14,12 +14,13 @@ jobs:
   analysis:
     name: Scorecards analysis
     runs-on: ubuntu-latest
+    if: github.event.repository.default_branch == github.ref_name
```

```suggestion
    if: github.ref_name == github.event.repository.default_branch
```

Nit: Logically the same, but I am used to putting the variable part first in a condition. 😉